freezer/freezer_url.go

```diff
@@ -0,0 +1,95 @@
+package freezer
+
+import (
+	"fmt"
+	"net/url"
+	"time"
+
+	"github.com/uw-labs/freezer"
+	"github.com/uw-labs/straw"
+	"github.com/uw-labs/substrate"
+	"github.com/uw-labs/substrate/suburl"
+)
+
+func init() {
+	suburl.RegisterSink("freezer+dir", newKafkaSink)
```

Comment:
Is there a reason to not support S3 sink?

Reply:
No, it's just not done yet because I haven't got around to it from what I remember.

Reply:
(basically didn't need it yet and don't have time to implement and test this week, but want to get the rest merged asap)

Reply:
ah ok 👍